grow/translators/base.py

```diff
@@ -97,7 +98,13 @@ def download(self, locales, save_stats=True, inject=False):
         langs_to_translations = {}
         new_stats = []
         def _do_download(lang, stat):
-            new_stat, content = self._download_content(stat)
+            try:
+                new_stat, content = self._download_content(stat)
+            except translator_errors.NotFoundError:
+                text = 'No translation source for {} locale.'
```

Comment:
Just a slight nit on the error message -- this should probably be something like `No translations to download for: {}`. Because the term "source" might be overloaded and confused with the source language, etc.